src/server/transport/pubsub/util.ts

```diff
@@ -0,0 +1,5 @@
+import type { PubSubChannelId } from './generic-pub-sub';
+
+export function globalChannelId(channelId: PubSubChannelId): string {
+  return `${channelId.namespace}-${channelId.value}`;
+}
```

Comment:
One more thought regarding the public interface. Both implementations are calling this `globalChannelId` helper everywhere they use the channel ID (often multiple times per method). Could the public interface methods just receive a `string` argument instead of `PubSubChannelId` and boardgame.io can take care of calling `globalChannelId`?

Reply:
I thought that namespacing the IDs would make the API more extensible in the future, avoiding id collisions and etc. But sure, replaced with string.